es6.md

```diff
@@ -75,9 +82,16 @@ A `yield` expression.
 ## ObjectPattern
 
 ```js
+interface PatternProperty <: Property {
+    type: "Property"; // inherited
+    value: Pattern;
+    kind: "init";
+    method: false;
```

Comment:
Like the overall idea, but WRT `.method` specifically in PatternProperty, couldn't it wait for #5 to be resolved first?